tools/nl/embeddings/build_embeddings.py

```diff
@@ -72,7 +75,7 @@ def _make_gcs_embeddings_filename(embeddings_size: str) -> str:
   minute_str = utils.two_digits(now.minute)
   second_str = utils.two_digits(now.second)
 
-  return f"embeddings_{embeddings_size}_{now.year}_{month_str}_{day_str}_{hour_str}_{minute_str}_{second_str}.csv"
+  return f"embeddings_{embeddings_size}_{model_version}_{now.year}_{month_str}_{day_str}_{hour_str}_{minute_str}_{second_str}.csv"
```

Comment:
nit: how about `embeddings_{embeddings_size}_{date_str}.{model_version}.csv` ?

So the embeddings size+version is not split, and `.` provides a convenient splitting point across files (like we do for final mode vs. int model)

Reply:
Done